app.py

```diff
@@ -12,6 +12,8 @@
 app.config.from_pyfile('settings.py',silent=True)
 if "VAULT_ADDR" in os.environ:
     app.config['VAULT_URL'] = os.environ['VAULT_ADDR']
+if "VAULT_SKIP_VERIFY" in os.environ:
```

Comment:
Yep, just like `VAULT_ADDR` also does override whatever the user stated in the settings file...
If you are so keen on removing these lines, you should do the same with the the previous ones. This way it would at least be consistent.